pallets/sto/src/benchmarking.rs

```diff
@@ -0,0 +1,246 @@
+#![cfg(feature = "runtime-benchmarks")]
+use crate::*;
+use frame_benchmarking::benchmarks;
+use frame_support::traits::Get;
+use frame_system::RawOrigin;
+use pallet_settlement::{
+    benchmarking::{add_transfer_manager, compliance_setup},
+    VenueDetails,
+};
+use polymesh_common_utilities::benchs::User;
+use polymesh_common_utilities::{asset::AssetType, benchs::UserBuilder};
+use polymesh_primitives::TrustedIssuer;
+
+const OFFERING_TICKER: Ticker = Ticker::repeating(b'A');
+const RAISE_TICKER: Ticker = Ticker::repeating(b'B');
+
+pub type Asset<T> = pallet_asset::Module<T>;
+pub type ComplianceManager<T> = pallet_compliance_manager::Module<T>;
+pub type Identity<T> = identity::Module<T>;
+pub type Timestamp<T> = pallet_timestamp::Module<T>;
+pub type Settlement<T> = pallet_settlement::Module<T>;
+pub type Sto<T> = crate::Module<T>;
+
+fn create_asset<T: Trait>(
```

Comment:
We've recently added `make_asset` & `make_ticker` to our `pallets/common/src/benchs/asset.rs`. 
You could reuse it if that fits into your use case.